Dockerfile.backend

```diff
@@ -20,17 +27,17 @@ WORKDIR /build/backend
 ARG VERSION
 ARG BUILD_TIMESTAMP
 ARG COMMIT_HASH
-ARG TARGETOS
-ARG TARGETARCH
 
 RUN --mount=type=cache,target=/go/pkg/mod \
     --mount=type=cache,target=/root/.cache/go-build \
```

Comment:
Cache mount is virtually ineffective in CI: https://github.com/moby/buildkit/issues/1512